static/js/topic_page/map_tile.tsx

```diff
@@ -275,14 +279,22 @@ function draw(
   const getTooltipHtml = (place: NamedPlace) => {
     let value = "Data Unavailable";
     if (place.dcid in chartData.dataValues) {
-      value = formatNumber(
-        Math.round((chartData.dataValues[place.dcid] + Number.EPSILON) * 100) /
-          100,
-        props.statVarSpec.unit
-      );
+      // shows upto 4 decimal points for very low values
```

Comment:
maybe we can use [toPrecision](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) & keep 2 significant digits for values whose absolute value is between 0-1.

Reply:
Done!